res/skins/Tango/skin_settings.xml

```diff
@@ -95,19 +118,121 @@ Description:
+                        <SetVariable name="ConfigKeyPerm">[Tango],always_visible_eject</SetVariable>
+                      </Template>
+
+                      <Template src="skin:button_2state_persist.xml">
+                        <SetVariable name="ObjectName">DeckButtonVisibility</SetVariable>
+                        <SetVariable name="Size">16f,15f</SetVariable>
+                        <SetVariable name="TooltipId">slip_mode</SetVariable>
+                        <SetVariable name="state_0_pressed">slipmode_off.svg</SetVariable>
+                        <SetVariable name="state_0_unpressed">slipmode_off.svg</SetVariable>
+                        <SetVariable name="state_1_pressed">slipmode_visible.svg</SetVariable>
+                        <SetVariable name="state_1_unpressed">slipmode_visible.svg</SetVariable>
+                        <SetVariable name="ConfigKeyPerm">[Tango],always_visible_slipMode</SetVariable>
+                      </Template>
+
+                      <Template src="skin:button_2state_persist.xml">
+                        <SetVariable name="ObjectName">DeckButtonVisibility</SetVariable>
+                        <SetVariable name="Size">16f,15f</SetVariable>
+                        <SetVariable name="TooltipId">repeat</SetVariable>
+                        <SetVariable name="state_0_pressed">repeat_off.svg</SetVariable>
+                        <SetVariable name="state_0_unpressed">repeat_off.svg</SetVariable>
+                        <SetVariable name="state_1_pressed">repeat_visible.svg</SetVariable>
+                        <SetVariable name="state_1_unpressed">repeat_visible.svg</SetVariable>
+                        <SetVariable name="ConfigKeyPerm">[Tango],always_visible_repeat</SetVariable>
+                      </Template>
+
+                      <Template src="skin:button_2state_persist.xml">
+                        <SetVariable name="ObjectName">DeckButtonVisibility</SetVariable>
+                        <SetVariable name="Size">15f,15f</SetVariable>
+                        <SetVariable name="TooltipId">orientation</SetVariable>
+                        <SetVariable name="state_0_pressed">cf_center.svg</SetVariable>
+                        <SetVariable name="state_0_unpressed">cf_center.svg</SetVariable>
+                        <SetVariable name="state_1_pressed">cf_visible.svg</SetVariable>
+                        <SetVariable name="state_1_unpressed">cf_visible.svg</SetVariable>
+                        <SetVariable name="ConfigKeyPerm">[Tango],always_visible_cfOrientation</SetVariable>
+                      </Template>
+
+                      <Template src="skin:button_2state_persist.xml">
+                        <SetVariable name="ObjectName">DeckButtonVisibility</SetVariable>
+                        <SetVariable name="Size">15f,15f</SetVariable>
+                        <SetVariable name="TooltipId">orientation</SetVariable>
```

Comment:
Thanks. I'll remove all tooltips from this section in skin settings since button labels are quite explicit.